cmd/minishift/cmd/delete.go

```diff
@@ -98,6 +100,28 @@ func removeInstanceConfigs() {
 	}
 }
 
+func removeProfileFromConfig() {
+	name := profileActions.GetProfileName()
+	activeProfile := profileActions.GetProfileName()
+
+	//If there are multiple profiles present
+	if name == activeProfile {
+		if !forceMachineDeletion {
+			fmt.Print("\nDo you want to delete the active profile? [y/N]: ")
+			fmt.Scanln(&confirm)
+			if strings.ToLower(confirm) != "y" {
+				atexit.Exit(1)
+			}
+		}
+	}
+	if name != "" {
```

Comment:
something in this order is not correct... only active profile need a confirmation? See #405

to keep current behaviour, it would be to delete and remove the active profile state. if you want confirmation, please do it across the board. This is else not really in line of a user's expectation.

Reply:
This is fixed in the latest code. Please check